wordpress-mobile/WordPress-iOS
# Comment Detail Redesign Info Button

Fixes #18999

To test:
1. Log in to a WP Account
2. In My Site, select Menu
3. Comments
4. Select any comment
5. Check if an info icon is placed next to user name.
6. Tap on the button and validate it opens a drawer with user information.
7. Tap on author URL and see if it opens the URL in a web view.

## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
